css/style.css

```diff
@@ -0,0 +1,364 @@
+body{
+  font-family: 'Monda', sans-serif;
```

Comment:
It could be better to add one more, common font (like Arial), just in case you favorite font somehow get lost.